src/containers/AccountConnection.jsx

```diff
@@ -260,23 +257,24 @@ class AccountConnection extends Component {
                       {t('account.config.security')}
                     </p>
                   }
-                </div>
+                </DescriptionContent>
             }
             {isSuccessTimedOut
               ? <div>
-                <h3>
-                  {t('account.connected.title', { name: connector.name })}
-                </h3>
-                <p>
-                  {t('account.connected.description', { name: connector.name })}
-                </p>
-                <p>
-                  {t('account.connected.ongoingSync', { name: connector.name })}
-                  <br />
-                  <span className={styles['bills-folder']}>
-                    {this.state.account.folderId}
-                  </span>
-                </p>
+                <DescriptionContent
+                  title={t('account.connected.title', { name: connector.name })}
+                  messages={[t('account.connected.description', { name: connector.name })]}
+                >
+                  {connector.dataType && connector.dataType.includes('bill') &&
```

Comment:
This message concerns only the `bill` datatype, so it won't be displayed it there are no bills to be synchronised. But we could add messages like this for other datatypes.